cmd/composeAndExecute.go

```diff
@@ -17,19 +17,19 @@
 package cmd
 
 import (
-	"github.com/ethereum/go-ethereum/statediff"
-	"github.com/vulcanize/vulcanizedb/libraries/shared/fetcher"
-	"github.com/vulcanize/vulcanizedb/libraries/shared/streamer"
-	"github.com/vulcanize/vulcanizedb/pkg/fs"
 	"os"
 	"plugin"
 	syn "sync"
 	"time"
 
+	"github.com/ethereum/go-ethereum/statediff"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 
+	"github.com/vulcanize/vulcanizedb/libraries/shared/fetcher"
+	"github.com/vulcanize/vulcanizedb/libraries/shared/streamer"
 	"github.com/vulcanize/vulcanizedb/libraries/shared/watcher"
+	"github.com/vulcanize/vulcanizedb/pkg/fs"
```

Comment:
If we're updating imports, I'm personally up for removing the spaces altogether so that it's not continually being thrown out of whack by GoLand

Reply:
Oh shoot I've been in the habit doing the opposite- going through and adding spaces between the different types of imports. I can revert to removing spaces.

Reply:
sorry to do an about face but I'm good with `goimports -w <dir>` as well 👍 

Should probably just add that to `make test` and CI to make sure it's applied uniformly, but that absolutely doesn't need to be part of this PR